src/Utility/ConstraintStabiliser.php

```diff
@@ -26,16 +47,16 @@ public static function stabiliseConstraints(OutputInterface $output, LibraryRele
         $parentLibrary = $releasePlanNode->getLibrary();
         $originalData = $composerData = $parentLibrary->getComposerData();
         $constraintType = $parentLibrary->getDependencyConstraint();
+        $parentName = $parentLibrary->getName();
 
         // Rewrite all dependencies.
         // Note: rewrite dependencies even if non-exclusive children, so do a global search
         // through the entire tree of the plan to get the new tag
-        $items = $releasePlanNode->getAllItems();
-        foreach ($items as $item) {
+        foreach (self::getAllItems($releasePlanNode) as $item) {
```

Comment:
What's the intention of this change?

Reply:
Issue was that a node was only looping its children when what it need to do is loop every every node in the plan.  It's to prevent this from happening https://github.com/silverstripe/recipe-collaboration/commit/a9f9ee5bdac56e18153db2e54d6575576d06c2ef ("silverstripe/recipe-cms": "5.x-dev" - instead of ~5.0.0@beta)